### PR TITLE
multi-stage build for compact docker image

### PR DESCRIPTION
**Pull request by aham-oudeis**
Co-Authored-By: Katherine Ebel <kathyebel@me.com>
Co-Authored-By: Nykaela Dodson <nykaelad@gmail.com>
Co-Authored-By: Hans Elde <hanserik.elde@gmail.com>

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
 .git
 .gitignore
-node_modules/
```
Comment (KatherineEbel) [inline .dockerignore:3]: why are we removing this?

File: Dockerfile [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+COPY --from=ts-builder ./home/backend_skopos/dist ./dist
+COPY --from=ts-builder ./home/backend_skopos/prisma ./prisma
+COPY package* ./
 
+RUN npm install --production
 
 EXPOSE 3001
 
-CMD ["npm", "run", "deploy"]
+CMD npx prisma migrate deploy; node ./dist/index.js
```
Comment (KatherineEbel) [inline Dockerfile:22]: Is there a reason for not running the deploy command?

Comment (aham-oudeis) [reply Dockerfile:22]: npm run deploy was failing in the container; if it works for you, no need to change

File: .dockerignore (hunk 1 above)
Comment (aham-oudeis) [reply .dockerignore:3]: without this, i was getting the typegraphql error; if it works without this, not needed

**Pull request closed without merging**